### ISSUE TITLE
SECURITY Microsoft Security Advisory CVE-2024-43483 and CVE-2024-43484

### ISSUE THREAD

**Issue by ibakirov** [ClosedXML/ClosedXML#2491]
## Read and complete the full issue template

Do not randomly delete sections. They are here for a reason.

**Do you want to request a *feature* or report a *bug*?**
- [x] Bug
- [ ] Feature
- [ ] Question

**Did you test against the latest CI build?**
- [ ] Yes
- [ ] No

If you answered `No`, please test with [the latest development build](https://ci.appveyor.com/project/ClosedXML/ClosedXML/branch/develop/artifacts) first.

**Version of ClosedXML**
0.104.1

**What is the current behavior?**
Current release 0.104.1 does not have updates for [CVE-2024-43483](https://github.com/advisories/GHSA-qj66-m88j-hmgj) and [CVE-2024-43484](https://github.com/advisories/GHSA-f32c-w444-8ppv)

**What is the expected behavior or new feature?**
New version of ClosedXML with updates for CVE-2024-43483 and CVE-2024-43484 mitigation.

**Is this a regression from the previous version?**

Regressions get higher priority. Test against the latest build of the previous minor version. For example, if you experience a problem on v0.95.3, check whether it the problem occurred in v0.94.2 too. 

## Reproducibility
**This is an important section. Read it carefully. Failure to do so will cause a 'RTFM' comment.**

Without a code sample, it is unlikely that your issue will get attention. Don't be lazy. Do the effort and assist the developers to reproduce your problem. Code samples should be [minimal complete and verifiable](https://stackoverflow.com/help/mcve). Sample spreadsheets should be attached whenever applicable. Remove sensitive information.

**Code to reproduce problem:**
```c#
public void Main()
{
    // Code standards:
    // - Fully runnable. I should be able to copy and paste this code into a 
    //   console application and run it without having to edit it much.
    // - Declare all your variables (this follows from the previous point)
    // - The code should be a minimal code sample to illustrate issue. The code 
    //   samples on the wiki are good examples of the terseness that I want. Don't
    //   post your full application.
}
```

- [ ] I attached a sample spreadsheet.  (You can drag files on to this issue)

**Comment by shapeh** [comment]
To elaborate - ClosedXML is linked to an outdated version of System.IO.Packaging - version 8.0.1 that has above-mentioned severe vulnerabilities.

**Comment by chillitom** [comment]
Note this issue has become more visible to people using the latest version of Visual Studio (17.12 / .NET 9). The built in NuGet package audit functionality now audits transitive dependencies by default where previously it defaulted to only direct dependencies.

**Comment by igitur** [comment]
Yet no-one dares to log a PR to fix this?

**Comment by chillitom** [comment]
In release 104.1 `System.IO.Packaging`  is at version `8.0.0` and shows as having vulnerabilites, however on the `develop` branch the `System.IO.Packaging` has been updated to 8.0.1 and no longer shows issues.

The `System.Net.Http` and `System.Text.RegularExpressions` libraries still appear to have problems.

![image](https://github.com/user-attachments/assets/866e52cf-d09c-418e-9f14-310d2cd02020)

The issue seems to be a transitive dependency introduced by the `RBush` package.

![image](https://github.com/user-attachments/assets/861cf086-0187-4442-bf49-a4cc62216556)

**Comment by chillitom** [comment]
> Yet no-one dares to log a PR to fix this?

Touché.

Taking a look.

**Comment by ibakirov** [comment]
@chillitom Thanks for RBush updates and I agree with ClosedXML upgrade for System.IO.Packaging 8.0.1 in develop branch (https://github.com/ClosedXML/ClosedXML/commit/2cf45186f005ee1d2a2a62286ed4d7b38763c4cb) from October 9th, but would like to see patch release or maybe 0.104.2.

**Comment by jahav** [comment]
I have released [0.104.2](https://github.com/ClosedXML/ClosedXML/releases/tag/0.104.2) that upgrades required references for System.IO.Packaging #2503 (through OpenXML SDK) and RBush one #2503.

I am pretty sure that `RBush` one is a false positive, but it doesn't really matter that much and even false positives need to be somehow solved.
